.ci/download-deps.sh

```diff
@@ -5,13 +5,36 @@ function get_deps() {
 
 	TSS_VERSION=2.3.1
 	ABRMD_VERSION=master
+	JSONC_VER=json-c-0.13.1-20180305
+
+    (apt-get update && apt-get --yes install expect) || \
+    dnf -y install expect || \
+    zypper --non-interactive install expect
```

Comment:
We would want to add these to the docker image, not to the script.

Reply:
You can modify the .docker files here and submit a PR:
https://github.com/tpm2-software/tpm2-software-container

Reply:
Will do. Just have to figure out how...

Reply:
@williamcroberts Ping (since I really have no idea about the docker stuff)